sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -68,6 +79,14 @@ def export(self, spans: Sequence[ReadableSpan], **kwargs: Any) -> SpanExportResu
         :rtype: ~opentelemetry.sdk.trace.export.SpanExportResult
         """
         envelopes = []
+        if spans and self._should_collect_otel_resource_metric():
+            resource = None
+            try:
+                tracer_provider = get_tracer_provider()
```

Comment:
Asked Osvaldo about this. Product was clear they want this tied to tracing and consider the scenario where a customer wants no tracing to be small and irrelevant. My guess is this is because a lot of the functionality for heartbeat and it's replacement is for things like the App Map which don't really do much without tracing.

Reply:
With the current implementation, the custom metric will be sent everytime the traceexporter exports. is this by design or should we be only sneding this once?